numpy/core/code_generators/generate_umath.py

```diff
@@ -791,8 +791,10 @@ def english_upper(s):
 'remainder':
     Ufunc(2, 1, None,
           docstrings.get('numpy.core.umath.remainder'),
-          None,
+          'PyUFunc_RemainderTypeResolver',
           TD(intflt),
+          [TypeDescription('m', FullTypeDescr, 'mm', 'm'),
+          ],
```

Comment:
Line wrapping here is a little weird, and doesn't match the other cases with only one `TypeDescription`